test/functional/cmdLineTests/criu/src/org/openj9/criu/JDK11UpTimeoutAdjustmentTest.java

```diff
@@ -62,6 +62,7 @@ private void test(String testName) throws InterruptedException {
 		case "testObjectWaitNotify":
 			testObjectWaitNotify();
 			break;
+/*
```

Comment:
i dont think we need changes on this file

Reply:
Its enough to just comment out the .xml

Reply:
Yeah, `criu_nonPortableJDK11Up.xml` change is sufficient.